d3ploy/demand_driven_deployment_inst.py

```diff
@@ -31,139 +31,141 @@ class DemandDrivenDeploymentInst(Institution):
     """
 
     facility_commod = ts.MapStringString(
-        doc="A map of facilities and each of their corresponding" +
-        " output commodities",
-        tooltip="Map of facilities and output commodities in the " +
-        "institution",
-        alias=['facility_commod', 'facility', 'commod'],
-        uilabel="Facility and Commodities"
+        doc="A map of facilities and each of their corresponding"
+        + " output commodities",
+        tooltip="Map of facilities and output commodities in the "
+        + "institution",
+        alias=["facility_commod", "facility", "commod"],
+        uilabel="Facility and Commodities",
     )
 
     facility_capacity = ts.MapStringDouble(
-        doc="A map of facilities and each of their corresponding" +
-        " capacities",
-        tooltip="Map of facilities and capacities in the " +
-        "institution",
-        alias=['facility_capacity', 'facility', 'capacity'],
-        uilabel="Facility and Capacities"
+        doc="A map of facilities and each of their corresponding"
+        + " capacities",
+        tooltip="Map of facilities and capacities in the " + "institution",
+        alias=["facility_capacity", "facility", "capacity"],
+        uilabel="Facility and Capacities",
```

Comment:
why add a comma? This is the final keyword argument item in the function call, no?

Reply:
I reverted all pep8 changes